.gitignore

```diff
@@ -7,3 +7,9 @@ src/encryption/__pycache__/asymmetric_key_pair.cpython-36.pyc
 src/encryption/__pycache__/encryption_controller.cpython-36.pyc
 src/encryption/__pycache__/key_data.cpython-36.pyc
 src/encryption/__pycache__/encoded_asymmetric_key_pair.cpython-36.pyc
+src/data/__pycache__/data_controller.cpython-37.pyc
+src/data/__pycache__/data_stream_controller.cpython-37.pyc
+src/encryption/__pycache__/asymmetric_key_pair.cpython-37.pyc
+src/encryption/__pycache__/encoded_asymmetric_key_pair.cpython-37.pyc
+src/encryption/__pycache__/encryption_controller.cpython-37.pyc
+src/encryption/__pycache__/key_data.cpython-37.pyc
```

Comment:
We can’t be adding compiled source(s) for every new file we import. Can you incorporate the `.gitignore` file like I asked you to in #2? Here’s a [sample](https://www.gitignore.io/api/python). Just do this in your terminal:

```shell
$ curl -L -s https://www.gitignore.io/api/python > .gitignore
```

... and we’re good to go (for the foreseeable future)

Reply:
Changes have been made.